gpio-adapter.js.js

```diff
@@ -66,6 +66,22 @@ class GpioProperty extends Property {
           this.device.notifyPropertyChanged(this);
         }
       });
+      if (this.device.pinConfig.momentaryButton) {
+        setTimeout(() => {
+          this.device.gpio.write(value = 0, (err) => {
+            if (err) {
+              console.error('GPIO: write for pin:', this.device.name, 'failed');
+              console.error(err);
+              reject(err);
```

Comment:
You're attempting to reject/resolve the same promise twice here. You can just omit the `reject(err)` here and the `resolve(this.value)` below.